elasticsearch/Dockerfile.centos7.s3repository

```diff
@@ -0,0 +1,66 @@
+FROM centos:centos7
```

Comment:
@richm @ewolinetz In general, I don't like the idea of adding a whole other Dockerfile.  Shouldn't we be able to build upon one that exists by adding env vars and args to one which already exists?

Reply:
> Shouldn't we be able to build upon one that exists by adding env vars and args to one which already exists?

Yes, we should

Reply:
I agree, I feel that if we have another Dockerfile that implies we fully support and will provide images for that Dockerfile.

If we provide the means to support custom images where a customer can specify and build with additional plugins I would prefer to go that route for now.